go.mod

```diff
@@ -21,12 +21,12 @@ require (
 	github.com/openconfig/goyang v0.2.7
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
 	github.com/pkg/errors v0.9.1
-	github.com/redhat-developer/app-services-sdk-go v0.9.4
+	github.com/redhat-developer/app-services-sdk-go v0.9.5-0.20210809114429-bb1f9a7aabe5
```

Comment:
Using the main branch is good for testing, but we should not do a release with it like that IMO.

Reply:
This was for testing purpose only. Will be changed before the `WIP` tag is removed.